calculatorUICOOP/Views/MainPage.xaml

```diff
@@ -33,30 +33,30 @@
+        <Button Clicked="Number_Clicked" Style="{StaticResource numButton}" Text="6" Grid.Row="3" Grid.Column="2"></Button>
 
-        <Button Clicked="Button_Clicked" Style="{StaticResource numButton}" Text="1" Grid.Row="4" Grid.Column="0"></Button>
-        <Button Clicked="Button_Clicked" Style="{StaticResource numButton}" Text="2" Grid.Row="4" Grid.Column="1"></Button>
-        <Button Clicked="Button_Clicked" Style="{StaticResource numButton}" Text="3" Grid.Row="4" Grid.Column="2"></Button>
+        <Button Clicked="Number_Clicked" Style="{StaticResource numButton}" Text="1" Grid.Row="4" Grid.Column="0"></Button>
+        <Button Clicked="Number_Clicked" Style="{StaticResource numButton}" Text="2" Grid.Row="4" Grid.Column="1"></Button>
+        <Button Clicked="Number_Clicked" Style="{StaticResource numButton}" Text="3" Grid.Row="4" Grid.Column="2"></Button>
 
-        <Button Clicked="Button_Clicked" Style="{StaticResource numButton}" Text="." Grid.Row="5" Grid.Column="0"></Button>
-        <Button Clicked="Button_Clicked" Style="{StaticResource numButton}" Text="0" Grid.Row="5" Grid.Column="1"></Button>
-        <Button Clicked="Button_Clicked" Style="{StaticResource numButton}" Text="=" Grid.Row="5" Grid.Column="2"></Button>
+        <Button Clicked="Decimal_Clicked" Style="{StaticResource numButton}" Text="." Grid.Row="5" Grid.Column="0"></Button>
+        <Button Clicked="Number_Clicked" Style="{StaticResource numButton}" Text="0" Grid.Row="5" Grid.Column="1"></Button>
+        <Button Clicked="Equals_Clicked" Style="{StaticResource numButton}" Text="=" Grid.Row="5" Grid.Column="2"></Button>
 
         <StackLayout Grid.RowSpan="4" Grid.Column="3" Grid.Row="2" BackgroundColor="LightGray" Padding="10" Margin="5">
-            <Button  FontSize="20" Text="C" CornerRadius="10" BackgroundColor="DimGray" TextColor="White"></Button>
-            <Button Style="{StaticResource opButton}" Text="/"></Button>
-            <Button Style="{StaticResource opButton}" Text="x"></Button>
-            <Button Style="{StaticResource opButton}" Text="+"></Button>
-            <Button Style="{StaticResource opButton}" Text="-"></Button>
-            <Button Style="{StaticResource opButton}" Text="%"></Button>
-            <Button Style="{StaticResource opButton}" Text="CE"></Button>
+            <Button Clicked="Delete_Clicked" FontSize="20" Text="C" CornerRadius="10" BackgroundColor="DimGray" TextColor="White"></Button>
+            <Button Clicked="Divide_Clicked" Style="{StaticResource opButton}" Text="/"></Button>
+            <Button Clicked="Multiply_Clicked" Style="{StaticResource opButton}" Text="x"></Button>
+            <Button Clicked="Plus_Clicked"   Style="{StaticResource opButton}" Text="+"></Button>
+            <Button Clicked="Minus_Clicked" Style="{StaticResource opButton}" Text="-"></Button>
+            <Button Clicked="Remainder_Clicked" Style="{StaticResource opButton}" Text="%"></Button>
```

Comment:
Not sure that Event Handler name accurately represents the functionality.
It's my understand it changes the current input to the % representation which is just the division of 100.